tests/managed/nestedClasses.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+namespace NestedClasses
+{
```

Comment:
Minor: [Style](http://www.mono-project.com/community/contributing/coding-guidelines/#where-to-put-braces)